src/rest-server/src/models/v2/job/k8s.js

```diff
@@ -151,6 +172,8 @@ const convertTaskDetail = async (taskStatus, ports, userName, jobName, taskRoleN
 
 const convertFrameworkDetail = async (framework) => {
   const completionStatus = framework.status.attemptStatus.completionStatus;
+  const diagnostics = completionStatus ? completionStatus.diagnostics : null;
+  const diagInfo = extractDiagnosticsInfo(diagnostics);
   const detail = {
```

Comment:
diagnostics may only contain controller reported msg, such as "Pod spec invalid: xxxx". So the msg is lost after this line.
Suggest to rename extractDiagnosticsInfo as extractPodCompletionStatus, and it should not be used as the whole diag.

Reply:
Will fix it